thirdeye/thirdeye-frontend/app/styles/components/button.scss

```diff
@@ -61,3 +61,10 @@
     }
   }
 }
+
+.btn span.glyphicon {
```

Comment:
### Blocking Comment: 
This selector is too general (it also affects the icons in the rca table). 
Please wrap this with a custom css class. Since we're overriding bootstrap, I'm okay with using !important (bs uses it everywhere anyway)